### PR TITLE
Add script to verify 1919 issue

### PR DESCRIPTION
**Pull request by HSavynetska**
ATF Test Scripts to check [#1919](https://github.com/SmartDeviceLink/sdl_core/issues/1919)

This PR is **ready** for review.

### Summary
A test script has been added to verify that the problem has been fixed.

### ATF version
develop

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+-- Steps to reproduce:
+-- 1) User activates non-media app_3
+--    HMI -> SDL: OnAppDeactivated ( <appID_2> )
+--    HMI -> SDL: SDL.ActivateApp ( <appID_3 )
+-- Expected result:
+-- 1) SDL -> navigation app_2 : OnHMIStatus (LIMITED, AUDIBLE)
+-- 2) SDL -> HMI: SDL.ActivateApp (SUCCESS)
+-- 3) SDL -> non-media app_3 : OnHMIStatus (FULL, NOT_AUDIBLE)
+-- Note: media app_1 still in LIMITED and AUDIBLE
+-- Actual result:
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua]: `Actula result` is redundant for script description

File: test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Applications Configuration ]]
+config.application1.registerAppInterfaceParams.appHMIType = { "MEDIA" }
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+config.application2.registerAppInterfaceParams.appHMIType = { "NAVIGATION" }
+config.application2.registerAppInterfaceParams.isMediaApplication = true
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua]: NAVI app should be with isMediaApplication = false

File: test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+config.application3.registerAppInterfaceParams.appHMIType = { "DEFAULT" }
+config.application3.registerAppInterfaceParams.isMediaApplication = false
+
+--[[ Local Functions ]]
+local function activateApp(pAppId)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua]: The better idea to not create a lot of similar functions and functions with a lot off conditions but create one table with expectations and use it in one single function.
Like you have function for app activation and send to it app id and current expectation:
```
local appsOnHMIStatusExp = {
mediaAppActivation = {
  navi = { ... }
  media = { ... }
  non-media = { .. }
}
}
local function appActivation(pAppID, pExpTbl)
  <activation messages>
  <OnHMIStatus expectation for media app>(pExpTbl.navi):Times(#pExpTbl.navi)
  <OnHMIStatus expectation for media app>(pExpTbl.media):Times(#pExpTbl.media)
  <OnHMIStatus expectation for media app>(pExpTbl.non-media):Times(#pExpTbl.non-media)
end
```

Please update script according to logic above.

File: test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+end
+
+local function activateNonMediaApp()
+  local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(3) })
+  EXPECT_HMIRESPONSE(requestId)
+  common.getMobileSession(3):ExpectNotification("OnHMIStatus", {hmiLevel = "FULL", audioStreamingState = "NOT_AUDIBLE" })
+end
+
+--[[ Test ]]
+runner.Title("Preconditions")
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua]: "MixingAudioSupported" should be set to true in .ini file

Comment (HSavynetska) [reply test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua]: @GetmanetsIrina You can find the updates in the commit cd2fa7e

File: test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua (hunk 3 above)
Comment (HSavynetska) [reply test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua]: @GetmanetsIrina You can find the updates in the commit cd2fa7e

File: test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua (hunk 2 above)
Comment (HSavynetska) [reply test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua]: @GetmanetsIrina You can find the updates in the commit cd2fa7e

File: test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua (hunk 1 above)
Comment (HSavynetska) [reply test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua]: @GetmanetsIrina You can find the updates in the commit cd2fa7e

**Comment by jacobkeeler** [comment]
Closing in favor of #2452

**Pull request closed without merging**